strapi/gcp-deployment/README.md

```diff
@@ -1,3 +1,44 @@
+# Notes on Strapi deployment
+
+## Deploying GCP infrastructure with Terraform
+
+### Setting up
+
+Ensure that `terraform` and `gcloud` are installed.
+
+Set your desired database username and password in `secret.tfvars`.
+
+If you already have a remote terraform state on GCS ready for use (as defined in `backend.tf`),
+simply run `terraform init` to set up terraform.
+Otherwise, set up terraform from scratch as follows:
+1. Comment out `backend.tf`
+2. Run `terraform init`: this initially sets up terraform with a local state
+3. Run `npm run terraform-deploy-init`: this will deploy a GCS bucket to hold your remote terraform state
```

Comment:
Ah yes, we'll have to change the backend because there's a cyclic dependency between creating the bucket the state is stored in and terraform creating the bucket. I'm wonder if specifying `--state=terraform.tfstate` will let us avoid this dance https://developer.hashicorp.com/terraform/language/settings/backends/local#state-filename (probably not, though).